what-is-chalk.mdx

```diff
@@ -18,215 +18,467 @@ import { PyDiffEditor } from '@/components/Editor'
 ```
 
-Chalk makes this data available for point-in-time-correct dataset
-generation for data science use cases. Every pipeline has built-in
-monitoring and alerting to ensure data quality and timeliness.
+These features are now available for low-latency [online inference](/docs/query-basics) and [offline training](/docs/training-client).
+The ability to re-use the same features in both contexts dramatically shortens development time and ensures that feature values from online and offline contexts match.
+
+---
+
+## Why Chalk?
+
+Chalk is the first platform to unify feature and prompt engineering, LLM evals, and real-time inference into a unified platform.
+Bridge the gap between experimentation and production with a single system that unifies offline and online inference.
+Chalk retrieves and computes dynamically at inference time with an execution engine called [Velox](https://github.com/facebookincubator/velox)--we maintain a fork that’s been heavily optimized for low-latency (< 3 ms).
+
+Unifying online and offline execution enables teams to
+
+- consolidate training and serving into a single version controlled environment
+- reduce time-to-production for new models from months to days
+- build advanced AI systems (RAG, RecSys, Fraud detection, etc.) without building custom infrastructure
+
+**No DSLs, no rewrites, just ordinary Python**
 
-### Reverse ETL
+Define features in Python that are run natively in Rust, C++, or as UDFs — no need to manually port notebooks and code into other languages e.g. Scala.
 
-When your model needs to use features that are canonically stored in
-a high-latency data source (like a data warehouse), Chalk's Reverse
-ETL support makes it simple to bring those features online and serve
-them quickly.
+**See everything, debug anything**
 
-Add a single line of code to an `offline` resolver, and Chalk constructs
-a managed reverse ETL pipeline for that data source:
+Traditional ML systems often operate as opaque black boxes, especially when transformation logic gets embedded across disparate data pipelines, obscuring the connection between inputs and predictions.
+Chalk disrupts this paradigm by bringing comprehensive observability to every aspect of your ML systems; easily
+
+- trace every model feature back to its original data sources with detailed lineage tracking — showing exactly where data came from
+- troubleshoot and optimize your ML pipelines with end-to-end tracing that tracks the inputs and outputs of every step in any run
+- monitor feature drift, access patterns, and performance metrics in real-time across all your model deployments
+
+Iterate faster and deploy with confidence — knowing exactly how your systems behave at each step of the machine learning lifecycle.
+
+---
+
+## Chalk for MLOps
+
+Let Chalk handle orchestrating, caching, and serving features at scale freeing up you and your team to focus on building models—not plumbing.
+
+Democratize modern software engineering practices, with Chalk
```

Comment:
my marketing brain goes off when i see democratize, but maybe that's just me :)

Reply:
Fixed@